tutorials/tts/NeMo_TTS_Primer.ipynb

```diff
@@ -2062,7 +2062,12 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.7"
+   "version": "3.9.15"
+  },
+  "vscode": {
+   "interpreter": {
+    "hash": "f8a1d50fd7b1e17bd198f085b8ced031398c6134b0da7c4415c17601bbcccc4e"
+   }
```

Comment:
nit: we may need to remove these to avoid conflicts every time when we update jupyter notes.